pkg/querier/blocks_store_queryable.go

```diff
@@ -676,25 +659,24 @@ func canBlockWithCompactorShardIndexContainQueryShard(queryShardIndex, queryShar
 	return true, false
 }
 
-func (q *blocksStoreQuerier) fetchSeriesFromStores(
-	ctx context.Context,
-	sp *storage.SelectHints,
-	clients map[BlocksStoreClient][]ulid.ULID,
-	minT int64,
-	maxT int64,
-	matchers []*labels.Matcher,
-	convertedMatchers []storepb.LabelMatcher,
-	maxChunksLimit int,
-	leftChunksLimit int,
-) ([]storage.SeriesSet, []ulid.ULID, storage.Warnings, int, error) {
+// fetchSeriesFromStores fetches series satisfying the given matchers (convertedMatchers) and belonging to the given time range (minT, maxT)
+// from all the given store-gateways (clients). Series are fetched from the given set of store-gateways concurrently.
```

Comment:
Nit on phrasing: I think it's better to just say `satisfying convertedMatchers and in the time range (minT, maxT)`.
(are both ends open? Maybe it's `[minT, maxT)`?)
Try to keep comments somewhere near 72 cols max - this one is 141.